src/commcare_cloud/commands/fab.py

```diff
@@ -62,6 +70,27 @@ def run(self, args, unknown_args):
             open(known_hosts_file, 'a').close()
         return exec_fab_command(args.env_name, *fab_args)
 
+    @staticmethod
+    def print_deploy_deprecation():
+        print(magenta('Hi.'))
+        print()
+        sleep(.5)
+        print(magenta('Things have changed.'))
+        print()
+        sleep(2)
+        print(cyan('The `commcare-cloud <env> fab deploy` command has been deprecated.'))
+        print()
+        sleep(3)
+        print(cyan('Instead, please use'))
+        print()
+        print(yellow('  commcare-cloud <env> deploy'))
+        sleep(4)
```

Comment:
:rofl:

Reply:
I'm mixed on this because I see what you're saying. I thought this worked well as a way to grab attention and have people read through the whole brief message the last time we went through one of these transitions (which was when we switched from running `fab <env> deploy` in the commcare-hq directory to using `commcare-cloud <env> deploy`). As not a very good reader myself, I imagined myself likely to absorb the message more quickly in the middle of a busy day if it's displayed to me like this versus just printed out into the screen at once (in which case I might run it again while flipping through tabs before realizing something was out of the ordinary and I should read it). So yeah I was going for engaging and slightly playful, but if instead it comes off as annoying and mildly patronizing or something like that then I'm happy to get rid of the pauses.

Reply:
I think we should be consistent about how we communicate these changes. That could mean a few things (some of these could be done together).

1. Write a changelog entry indicating that it will change soon
2. `fab deploy` tells you to use `deploy` and exits
3. `fab deploy` tells you before the command is run that it will be deprecated soon and you should use `deploy` in the future, but continues to deploy as normal
4. `fab deploy` works as it does now and tells you to run `deploy` in the future once deploy is complete

I think if I were trying to run this as a third party and not keeping up with the github PRs, then I'd expect a changelog entry and it'd be nice to have 2, 3 and/or 4 (probably either 3 with a prompt (`you know this is old right? y/n`) or 4 as the nicest experience). I don't think I'd expect something reminiscent of [sl](https://github.com/mtoyoda/sl)

Reply:
My $0.02: I also found this decision a bit strange, if kinda cute. The slow output over 10s would not indicate to me that I should read carefully and then do something different. I rarely look output of deploy commands until after they have completed, and my terminal clearly tells me when the script has exited so it would be easier for me to spot that something went wrong if it printed the error and exited immediately.